osquery/tables/system/windows/certificates.cpp

```diff
@@ -322,7 +581,24 @@ void enumerateCertStore(const HCERTSTORE& certStore,
   auto certContext = CertEnumCertificatesInStore(certStore, nullptr);
 
   if (certContext == nullptr && GetLastError() == CRYPT_E_NOT_FOUND) {
-    TLOG << "    Store was empty.";
+    VLOG(1) << "Store was empty.";
+
+    // Personal stores for other users come back as empty, even if they are not.
+    auto is_personal_store = storeName == "Personal" && !username.empty();
+    // Avoid duplicate rows for personal certs we've already inserted up
+    // front.
+    auto not_already_added =
+        storeLocation != "Users" || boost::ends_with(storeId, "_Classes");
+
+    if (is_personal_store && not_already_added) {
+      VLOG(1) << "Trying harder to get Personal store.";
```

Comment:
Hrm, what are your thoughts on the verbiage of this? if I saw this message coming across in my logs, I'd be curious what `Trying harder` means. How would you feel about elaborating a _bit_ more? Perhaps something to the affect of `Checking disk for additional user stored certificates` or something? 

Another thought I had, is there any extra added cost to "trying harder"? Is this something that we should be toggling on/off in the event someone wants more performance?